roles/https_haproxy/tasks/main.yml

```diff
@@ -32,6 +32,21 @@
     - restart haproxy
   when: enable_https
 
+- name: Copy ocsp script
+  copy:
+    src: ocsp.sh
+    dest: /usr/local/bin/ocsp.sh
```

Comment:
I'd prefer to keep all SCZ-related files under /opt.
Maybe we should introduce `/opt/scz-misc/bin` for these kind of things?